workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Jobs/CodegenJob.cs

```diff
@@ -76,10 +76,15 @@ public bool IsDirty()
             var schemaFiles = InputFiles
                 .Select(file =>
                 {
+                    if (detailsStore?.FileTree == null)
```

Comment:
I don't think this is the correct way to solve this, shouldn't we instead mock out the file tree? 

This introduces a new codepath that actual running codegen code could take if I'm understanding this correctly